packages/myst-frontmatter/src/exports/exports.yml

```diff
@@ -256,3 +256,39 @@ cases:
           articles:
             - a.md
     errors: 3
+  - title: pdf filename coerces to format and output
+    raw:
+      exports:
+        - out.pdf
+    normalized:
+      exports:
+        - format: pdf
+          output: out.pdf
+  - title: pdf extension coerces to format
+    raw:
+      exports:
+        - .pdf
+    normalized:
+      exports:
+        - format: pdf
+  - title: pdf output adds format
+    raw:
+      exports:
+        - output: out.pdf
+    normalized:
+      exports:
+        - format: pdf
+          output: out.pdf
+  # - title: export with only template validates
+  #   raw:
+  #     exports:
+  #       - template: template
+  #   normalized:
+  #     exports:
+  #       - template: template
```

Comment:
skipped test?

Reply:
Yes - this is looking ahead to when we infer format from template (and therefore only template would be ok). Not implemented yet!